notebooks/MosvizExample.ipynb

```diff
@@ -59,24 +59,71 @@
     "    elif 's2d' in file_path:\n",
     "        spectra_2d.append(file_path)\n",
     "        \n",
-    "        \n",
     "for i in range(len(spectra_1d)):\n",
-    "    images.append(str(level3_path / 'mosviz_cutouts' / '227.fits'))\n",
+    "    images.append(str(level3_path / 'mosviz_cutouts' / '227.fits'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosviz.load_data(spectra_1d, spectra_2d, images=images)\n",
```

Comment:
Just a note that you can still use the old syntax here, you don't necessarily have to name the images argument. Maybe we actually want to keep it as-is (I say, having already approved the PR)?

Reply:
On the other hand, writing it explicitly ensures the user gets the point that it's an optional argument.

Reply:
Good point! Merging...